src/test/java/com/intel/gkl/testing_utils/TestingUtils.java

```diff
@@ -0,0 +1,38 @@
+package com.intel.gkl.testing_utils;
+
+import java.security.SecureRandom;
+import java.util.Arrays;
+
+public class TestingUtils {
+    public static final int SIZE_4MB = 4 *1024 * 1024;
+    public static final int MAX_ARRAY_SIZE = Integer.MAX_VALUE -2;
```

Comment:
Can you add note for why we sub 2 here

Reply:
Good point, it might be specific to the JDK or even JVM. This is the biggest possible value for an array size in our current Java environment, which is openjdk-1.8. I can add a comment on that.